setup.py

```diff
@@ -118,11 +119,7 @@ def find_rel_test():
     platforms="Linux",
     test_suite="test.framework.suite",
     zip_safe=False,
-    install_requires=[
-        'setuptools >= 0.6',
-        "vsc-install >= 0.9.19",
-        "vsc-base >= 2.5.8",
-    ],
+    install_requires=['setuptools >= 0.6'],
```

Comment:
so where are we still using it?

Reply:
`setuptools` is still uses for now, yes, that needs more work